includes/ical-generator-functions.php

```diff
@@ -65,6 +66,10 @@ function generate_event( $post ) {
 		$location      = "{$slack_channel} channel on Slack";
 	}
 
+	if ( $wptv_url ) {
+		$description .= "WPTV URL link: {$wptv_url}\\n";
```

Comment:
I see URL and link has having the same meaning, removing URL makes it consistent with Slack below 
```suggestion
		$description .= "WPTV link: {$wptv_url}\\n";
```

Reply:
Thanks. It makes more sense to change to that.